HW__2.postman_collection.json

```diff
@@ -0,0 +1,238 @@
+{
+	"info": {
+		"_postman_id": "b531c509-1551-490e-a1a3-d6e2577403e2",
+		"name": "HW_2",
+		"schema": "https://schema.getpostman.com/json/collection/v2.1.0/collection.json",
+		"_exporter_id": "21168131"
+	},
+	"item": [
+		{
+			"name": "HW_2.3",
+			"event": [
+				{
+					"listen": "test",
+					"script": {
+						"exec": [
+							"//task 1\r",
+							"pm.test(\"Status code is 200\", function () {\r",
+							"    pm.response.to.have.status(200);\r",
+							"});\r",
+							"pm.test(\"Check the Content-Type\", function () {\r",
+							"    pm.expect(pm.response.headers.get('Content-Type')).to.eql(\"application/json; charset=utf-8\");\r",
+							"});\r",
+							"//Check the response to find the \"Morty Smith\" character info > find the character id.\r",
+							"const jsonObj = pm.response.json();\r",
+							"const name1= jsonObj.results.find(({name}) => name === \"Morty Smith\");\r",
+							"console.log(\"ID of Morty Smith it is \" + name1.id);\r",
+							"\r",
+							"\r",
+							"// the character id> \"Morty Smith\" +1\r",
+							"pm.sendRequest(\"https://rickandmortyapi.com/api/character/get\",function(t){\r",
```

Comment:
Not correct query. Please, review the [documentation](https://rickandmortyapi.com/documentation/#get-a-single-location) how to get a single character.